server/package.json

```diff
@@ -60,7 +60,7 @@
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "cookie-parser": "^1.4.6",
-    "exiftool-vendored": "26.0.0",
+    "exiftool-vendored": "^28.1.0",
```

Comment:
Could we please fix and re-enable the e2e test for the motion photos before this gets merged in?